<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>animals</class>
 <widget class="QWidget" name="animals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>178</width>
     <height>158</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                 ;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(168, 237, 234, 255), stop:1 rgba(180, 144, 202, 255));
       border-style: inset;
    border-width:4px;
 border-radius:40px;
   border-color: black;
    max-width:150px;
   max-height:150px;
  min-width:170px;
 min-height:150px;
                                 }
 QPushButton:hover{
 border: 4px inset black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(48, 207, 208, 255), stop:1 rgba(51, 8, 103, 255));
 }
                            </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>178</width>
     <height>158</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                 ;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(168, 237, 234, 255), stop:1 rgba(180, 144, 202, 255));
       border-style: inset;
    border-width:4px;
 border-radius:40px;
   border-color: black;
    max-width:150px;
   max-height:150px;
  min-width:170px;
 min-height:150px;
                                 }
 QPushButton:hover{
 border: 4px inset black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(48, 207, 208, 255), stop:1 rgba(51, 8, 103, 255));
 }
                            </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>178</width>
     <height>158</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                 ;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(168, 237, 234, 255), stop:1 rgba(180, 144, 202, 255));
       border-style: inset;
    border-width:4px;
 border-radius:40px;
   border-color: black;
    max-width:150px;
   max-height:150px;
  min-width:170px;
 min-height:150px;
                                 }
 QPushButton:hover{
 border: 4px inset black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(48, 207, 208, 255), stop:1 rgba(51, 8, 103, 255));
 }
                            </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Script</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-style: outset;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(168, 237, 234, 255), stop:1 rgba(180, 144, 202, 255));
border-color: green;
border-width:2px;


                                             
 </string>
   </property>
   <property name="text">
    <string>animals</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>30</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="midLineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>Select an image by name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
